apps/web/src/components/tx/ConfirmTxDetails/TxDetails.tsx

```diff
@@ -157,6 +152,21 @@ export const TxDetails = ({ safeTx, txData, showHashes }: TxDetailsProps) => {
                 </TxDetailsRow>
 
                 <TxDetailsRow label="Nonce">{safeTx.data.nonce}</TxDetailsRow>
+
+                <Button onClick={() => setShowHashes(!showHashes)} sx={{ all: 'unset' }}>
+                  <Typography
+                    variant="body2"
+                    fontWeight={700}
+                    display="inline-flex"
+                    alignItems="center"
+                    color="primary.light"
+                  >
+                    Show transaction hashes{' '}
```

Comment:
```suggestion
                    Transaction hashes{' '}
```

Makes more sense in the noun form.

Reply:
Changed in 8088f81565c50bbda19222d8b920bd696b365acd.